fiftyone/factory/repos/execution_store.py

```
@@ -207,6 +208,31 @@ def list_keys(self, store_name: str) -> List[str]:
+        Args:
+            store_name: the name of the store
+            key_prefix: optional key prefix to filter by (e.g. ``"run:"``)
+            exclude_fields: optional list of top-level fields to exclude
+                from each value dict (e.g. ``["result_ids"]``). The
+                exclusion is applied at the database level so heavy
+                fields are never transferred over the wire.
+
+        Returns:
+            a list of value dicts
+        """
+        pass
```

Ruff (0.15.9)

[warning] Unnecessary `pass` statement

Remove unnecessary `pass`

(PIE790)